packages/interop-test-client/src/index.spec.ts

```diff
@@ -6,9 +6,10 @@ import { spawnSync } from "node:child_process";
 import { Server } from "node:http";
 
 const JANUS_INTEROP_AGGREGATOR_IMAGE =
-  "us-west2-docker.pkg.dev/divviup-artifacts-public/janus/janus_interop_aggregator:0.5.2@sha256:15394dcc4d6eacf4e2112ef0d15a972328a7d805cb39ba9fc607b6a3e62fa9df";
+  "us-west2-docker.pkg.dev/divviup-artifacts-public/janus/janus_interop_aggregator@sha256:8cc873f7a8be459fe2dbecdf78561806b514ac98b4d644dc9a7f6bb25bb9df02";
```

Comment:
```suggestion
  "us-west2-docker.pkg.dev/divviup-artifacts-public/janus/janus_interop_aggregator:0.6.3@sha256:8cc873f7a8be459fe2dbecdf78561806b514ac98b4d644dc9a7f6bb25bb9df02";
```

Reply:
(sorry, I fat-fingered "add single comment" when I meant to cancel a comment) Could you do the same on the image below?